xen/arch/arm/cpufreq/wrappers.h

```diff
@@ -0,0 +1,239 @@
+/*
+ * xen/arch/arm/cpufreq/wrappers.h
```

Comment:
BTW, these wrappers are common for all code ported from Linux. Later on it would be great to make this file generic.

Reply:
yes.